samples/MvcSample.Web/Startup.cs

```diff
@@ -16,15 +16,75 @@
 #if DNX451
 using Microsoft.Framework.DependencyInjection.Autofac;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
 #endif
+using System.ComponentModel;
 using MvcSample.Web.Filters;
 using MvcSample.Web.Services;
 
-
 namespace MvcSample.Web
 {
     public class Startup
     {
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            services.AddCaching();
+            services.AddSession();
+
+            services.AddMvc();
+            services.AddSingleton<PassThroughAttribute>();
+            services.AddSingleton<UserNameService>();
+            services.AddTransient<ITestService, TestService>();
+
+            services.ConfigureMvc(options =>
+            {
+                options.Filters.Add(typeof(PassThroughAttribute), order: 17);
+                options.AddXmlDataContractSerializerFormatter();
+                options.Filters.Add(new FormatFilterAttribute());
+            });
+
+#if DNX451
+            // REVIEW: This is hella hacky
```

Comment:
- replace this comment with a copy of the one in `Configure(IApplicationBuilder app)`
- this is not at all hacky compared to the previous approach of overriding the `static IServiceProvider` in all functional tests
- must remain as is until aspnet/Configuration#159 is fixed